src/spikeinterface/sortingcomponents/benchmark/tests/common_benchmark_testing.py

```diff
@@ -21,12 +21,6 @@
 ON_GITHUB = bool(os.getenv("GITHUB_ACTIONS"))
 
 
-if hasattr(pytest, "global_test_folder"):
```

Comment:
Let's keep this one, because is imported somewhere else. 

Or you need to remove the import from `common_benchmark_testing`